plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js

```diff
@@ -19,6 +19,11 @@ function initTabs(){
             showCorrespondingTabBody(element)
             element.addEventListener('click', (event) => toggleSections(event))
         })
+    let cached = localStorage.getItem("active-tab")
+    if (cached !== null) {
+        let parsed = JSON.parse(cached)
+        document.querySelector("div[tabs-section] > button[data-togglable=" + parsed + "]").click()
```

Comment:
Small thing but this will cause an error when on the first page we click on a tab and then move to another page that doesn't have this tab.

Reply:
Fixed